.travis.yml

```diff
@@ -0,0 +1,2 @@
+script:
+  - rbenv global 2.3.1 && gem install ministryofjustice-danger --version '~> 0.1' && danger
```

Comment:
What does this actually do? At the moment all PRs result in a failed Travis build because there's no Rakefile. Should there be a Rakefile? What would be in it?